keras_cv/models/backbones/efficientnet_lite/efficientnet_lite_backbone.py

```diff
@@ -0,0 +1,368 @@
+
+        # Build stem
+        x = img_input
+
+        if include_rescaling:
+            # Use common rescaling strategy across keras_cv
+            x = keras.layers.Rescaling(1.0 / 255.0)(x)
+
+        x = keras.layers.ZeroPadding2D(
+            padding=utils.correct_pad_downsample(x, 3), name="stem_conv_pad"
+        )(x)
+        x = keras.layers.Conv2D(
+            32,
+            3,
+            strides=2,
+            padding="valid",
+            use_bias=False,
+            kernel_initializer=conv_kernel_initializer(),
+            name="stem_conv",
+        )(x)
+        x = keras.layers.BatchNormalization(axis=BN_AXIS, name="stem_bn")(x)
+        x = keras.layers.Activation(activation, name="stem_activation")(x)
+
+        # Build blocks
+        block_id = 0
+        blocks = float(sum(stackwise_num_repeats))
+
+        pyramid_level_inputs = []
+
+        for i in range(len(stackwise_kernel_sizes)):
+            num_repeats = stackwise_num_repeats[i]
+            input_filters = stackwise_input_filters[i]
+            output_filters = stackwise_output_filters[i]
+            # Update block input and output filters based on depth multiplier.
+            input_filters = round_filters(
+                filters=input_filters,
+                width_coefficient=width_coefficient,
+                depth_divisor=depth_divisor,
+            )
+            output_filters = round_filters(
+                filters=output_filters,
+                width_coefficient=width_coefficient,
+                depth_divisor=depth_divisor,
+            )
+
+            if i == 0 or i == (len(stackwise_kernel_sizes) - 1):
+                repeats = num_repeats
+            else:
+                repeats = round_repeats(
+                    repeats=num_repeats,
+                    depth_coefficient=depth_coefficient,
+                )
+            strides = stackwise_strides[i]
+
+            for j in range(repeats):
+                # The first block needs to take care of stride and filter size
+                # increase.
+                if j > 0:
+                    strides = 1
+                    input_filters = output_filters
+
+                if strides != 1:
+                    pyramid_level_inputs.append(utils.get_tensor_input_name(x))
+
+                # 97 is the start of the lowercase alphabet.
+                letter_identifier = chr(j + 97)
+                x = apply_efficient_net_lite_block(
+                    inputs=x,
+                    filters_in=input_filters,
+                    filters_out=output_filters,
+                    kernel_size=stackwise_kernel_sizes[i],
+                    strides=strides,
+                    expand_ratio=stackwise_expansion_ratios[i],
+                    activation=activation,
+                    dropout_rate=drop_connect_rate * block_id / blocks,
+                    name="block{}{}_".format(i + 1, letter_identifier),
```

Comment:
Shouldn't the underscore be between the parens?

Reply:
Umm, I think it is correct. In this [colab](https://colab.research.google.com/drive/1KAmySQYa2VT1CjMJePkk9h4FFFzzKNMc?usp=sharing), the naming are matching.